src/Jsonize.Test/project.json

```diff
@@ -5,9 +5,7 @@
   },
 
   "dependencies": {
-    "Jsonize": {
-        "target":  "project"
-    },
+  	"jsonize": "1.0.2",
```

Comment:
This is an old version. Please use the version 

"Jsonize": {
"target":  "project"		
 }

This should grab the latest build of Jsonize from it's project folder. You might need to re-clone the project if it's not working.